crates/bevy_gizmos/src/config.rs

```diff
@@ -178,25 +178,29 @@ impl Default for GizmoConfig {
             line_perspective: false,
             line_style: GizmoLineStyle::Solid,
             depth_bias: 0.,
+            #[cfg(feature = "bevy_render")]
             render_layers: Default::default(),
 
             line_joints: GizmoLineJoint::None,
         }
     }
 }
 
+#[cfg(feature = "bevy_render")]
 #[derive(Component)]
 pub(crate) struct GizmoMeshConfig {
     pub line_perspective: bool,
     pub line_style: GizmoLineStyle,
-    pub render_layers: RenderLayers,
+    pub render_layers: bevy_render::view::RenderLayers,
 }
 
+#[cfg(feature = "bevy_render")]
 impl From<&GizmoConfig> for GizmoMeshConfig {
     fn from(item: &GizmoConfig) -> Self {
         GizmoMeshConfig {
             line_perspective: item.line_perspective,
             line_style: item.line_style,
+            #[cfg(feature = "bevy_render")]
```

Comment:
Is this necessary if we're already inside of an identical cfg block?

Reply:
Nice catch, fixed.